### DISCUSSION TITLE
Beta testing in Google Play vs Firebase App Tester?

### DISCUSSION THREAD

**Discussion by RicoElectrico** [organicmaps/organicmaps#5013] [Q&A] [answered]
Hi, so I am a little bit confused about that. Google apparently has two parallel systems for deploying beta version of APKs:
- in Google Play (e.g. StreetComplete)
- via Firebase App Tester (currently used by Organic Maps)
How do they differ? Firebase App Tester is weird - a Google product that needs to be side-loaded (!)

**Comment by biodranik** [comment, marked as answer]
The main reason why we don't use Google Play betas is that all users' reviews from betas are not reflected in the production application. It means that if thousands of happy beta testers will rate the app, their ratings won't influence the production version.

We plan to make betas more flexible in the future, now there's not much time to do everything perfectly.
